express-serve-static-core/index.d.ts

```diff
@@ -1096,10 +1096,12 @@ interface Application extends IRouter, Express.Application {
      * Used to get all registered routes in Express Application
      */
     _router: any;
+
+    use(RequestHandler): this;
```

Comment:
I had to change types in IRouter.use to 
```
use: IRouterHandler<this> | IRouterMatcher<this>;
```
instead of 
```
use: IRouterHandler<this> & IRouterMatcher<this>;
```
to have ability to extend it using interfaces, e.g.:
```
use(...handlers: RequestHandlerParams[]): any;
```